docs/src/guide.md

```diff
@@ -0,0 +1,58 @@
+
+Before the pull request is merged, retriggering the Registrator bot on the same package (via a new comment or use of the web interface) will cause the bot to update the pull request (by force pushing).
+
+While usually starting the registration process is all the package author needs to do, occasionally, however, something more may be needed. In this case, the "waiting for author" label will be applied to the pull request, and a maintainer will comment on the pull request. Several common scenarios and what to do are listed below; these are meant to be linked to help explain a particular situation when it arises.
+
+## Compatability ("needs compat")
+
+Every Julia package has at least one (implicit) dependency: the Julia language itself. It is recommended that for every dependency, each package declares what versions of the dependency it is compatible with. This is done in the package's `Project.toml` file, and is documented [here](https://julialang.github.io/Pkg.jl/v1/compatibility/) in the `Pkg` standard library's doucmentation.
+
+Packages should at least declare the Julia version they are compatible with, by e.g.
+
+```toml
+[compat]
+julia = "1"
+```
+
+which declares that the package is compatible with all Julia versions from 1.0 onwards, until but not including, a possible 2.0 in the future.
+
+Tt is recommended to have one entry in the `compat` section of `Project.toml` for every dependency, not just Julia itself. Note that standard libraries are currently coupled to the Julia version, and don't have separate version numbers. So while they are declared as dependencies, they don't need separate entries in the `compat` section from Julia itself.
+
+## Merge conflicts (i.e. "needs rebase")
+
+If the pull request against the General registry has merge conflicts with the General registry itself, the maintainers of the General registry will be unable to merge the pull request. In this case, they will add the "waiting for author" tag and comment that there are merge conflicts or that the pull request needs a rebase. Just retrigger the Registrator.jl bot (either add another comment with "@JuliaRegistrator register", or use the web interface), and it will update the pull request.
+
+## Julia Package naming conventions
+
+The guidelines for naming packages are listed [here](https://julialang.github.io/Pkg.jl/v1/creating-packages/#Package-naming-guidelines-1) in the `Pkg` standard library's documentation. If a new package's name does not adhere to those guidelines or could otherwise cause confusion, maintainers of the General registry or the Julia community might comment on the pull request to ask that the name be changed or why the usual conventions were not used in this case. 
```

Comment:
```suggestion
The guidelines for naming packages are listed [here](https://julialang.github.io/Pkg.jl/v1/creating-packages/#Package-naming-guidelines-1) in the package manager's documentation. If a new package's name does not adhere to those guidelines or could otherwise cause confusion, maintainers of the General registry or the Julia community might comment on the pull request to ask that the name be changed, or why the usual conventions were not used in this case.